### ISSUE TITLE
External Reading the path and parameter file

### ISSUE THREAD

**Issue by wjlei1990** [rmodrak/asdf_examples#1]
(Maybe I am not right)

Is this directory an example or if you want to make it into script and run in production?

If it is production script, I may suggest to put the path and parameter file into external files that could be passed by system args. Use the `argparse` library as we did in `pypaw`

**Comment by rmodrak** [comment]
Good point, these are examples rather than production scripts, since it seemed that's what Carl needed most.

With readable examples, my hope is it becomes very easy to create production scripts.   I agree, `argparse` is nice to work with.

Thanks for the feedback.
